lib/WP_Auth0_LoginManager.php

```diff
@@ -618,21 +619,18 @@ private function clean_id_token( $id_token_obj ) {
 	/**
 	 * Generate the Auth0 logout URL.
 	 *
-	 * @param null $return_to - Site URL to return to after logging out.
+	 * @param string|null $return_to - Site URL to return to after logging out.
```

Comment:
Should this be string|null ? null is the default if no string is passed. The intent is to be passed a string? Maybe this is a PHP thing? as I see it all over the place

Reply:
We accept null in this case (although any falsy value would technically work, null is just the standard) because in the event none is provided we'll just [use the value returned from the home_url() function](https://github.com/auth0/wp-auth0/pull/814/files/6a54147cd9959efaf87dfd9a380129928459cc79#diff-8b3a370bef1a7697a84121a43cc266867da01dd4c58ec61c238fea8157237126R633)